src/nxdoc/nuxeo-server/upgrading-the-nuxeo-platform/how-to-upgrade-from-lts-2023-to-lts-2025/how-to-upgrade-audit-service.md

```diff
@@ -0,0 +1,254 @@
+will enable the OpenSearch Audit Backend. The previous `audit.elasticsearch.enabled` configuration property has no more effect.
+
+Previously, the ElasticSearch / OpenSearch configuration property for Audit looked like:
+
+```properties
+# generic
+elasticsearch.addressList=http://localhost:9200
+elasticsearch.index.translog.durability=request
+elasticsearch.indexNumberOfShards=5
+elasticsearch.indexNumberOfReplicas=1
+elasticsearch.restClient.connectionTimeoutMs=30000
+elasticsearch.restClient.socketTimeoutMs=121000
+elasticsearch.restClient.ssl.certificate.verification=true
+
+# optional
+elasticsearch.restClient.username=
+elasticsearch.restClient.password=
+elasticsearch.restClient.truststore.path=
+elasticsearch.restClient.truststore.password=
+elasticsearch.restClient.truststore.type=
+elasticsearch.restClient.keystore.path=
+elasticsearch.restClient.keystore.password=
+elasticsearch.restClient.keystore.type=
+
+# audit specific
+audit.elasticsearch.indexName=${elasticsearch.indexName}-audit
+audit.elasticsearch.latestLogId.afterDate=now-14d/d
+```
+
+Now you will configure OpenSearch / ElasticSearch connection with:
+
+```properties
+# generic
+nuxeo.opensearch1.client.server=http://localhost:9200
+
+nuxeo.opensearch1.client.connectionTimeout=3s
+nuxeo.opensearch1.client.socketTimeout=121s
+nuxeo.opensearch1.client.sslCertificateVerification=true
+
+# optional
+nuxeo.opensearch1.client.username=
+nuxeo.opensearch1.client.password=
+nuxeo.opensearch1.client.trustStore.path=
+nuxeo.opensearch1.client.trustStore.password=
+nuxeo.opensearch1.client.trustStore.type=
+nuxeo.opensearch1.client.keyStore.path=
+nuxeo.opensearch1.client.keyStore.password=
+nuxeo.opensearch1.client.keyStore.type=
+
+# audit specific
+nuxeo.audit.backend.default.opensearch1.index.name=nuxeo-audit
+nuxeo.audit.backend.default.opensearch1.settings.indexTranslogDurability=request
+nuxeo.audit.backend.default.opensearch1.settings.numberOfShards=5
+nuxeo.audit.backend.default.opensearch1.settings.numberOfReplicas=1
+nuxeo.audit.opensearch1.latestLogId.afterDate=now-14d/d
+```
+
+### Audit in MongoDB
+
+To store the Nuxeo audit in MongoDB you need to install the [nuxeo-audit-mongodb](https://connect.nuxeo.com/nuxeo/site/marketplace/package/nuxeo-audit-mongodb)
+package.
+After installation, check that the `mongodb-audit` template is included in your `nuxeo.conf` file. The template
+will enable the MongoDB Audit Backend.
+
+The package works with your `nuxeo.mongodb.*` configuration properties, as before. The only property to update is:
+`nuxeo.mongodb.audit.collection.name`, see new configuration properties:
+
+```properties
+nuxeo.audit.backend.default.mongodb.collection.name=audit
+
+# all nuxeo.mongodb.* properties are available for audit backend, for instance
+nuxeo.audit.backend.default.mongodb.dbname=nuxeo
+```
+
+The extended info are now stored as Object in MongoDB instead of plain String. This allows to enable the feature 
+to search in extended infos.
```

Comment:
```suggestion
The extended info is now stored as an Object in MongoDB instead of a plain String. This enables the search in extended info feature.
```